rust/flatc/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "flatc"
+version = "0.1.0"
+authors = ["Richard Berry <rjsberry@pm.me>", "FlatBuffers Maintainers"]
+license = "Apache-2.0"
+edition = "2018"
+build = "build.rs"
+description = "flatc Rust build script wrapper"
```

Comment:
I think we need to make it clear that this new Cargo package is only part of the story -- how about making this description read: `Companion package to flatbuffers, to generate code at compile time.`?